src/views/thingstotry/thingstotry.jsx

```diff
@@ -0,0 +1,90 @@
+var React = require('react');
+var injectIntl = require('react-intl').injectIntl;
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var render = require('../../lib/render.jsx');
+
+var MasonryGrid = require('../../components/masonrygrid/masonrygrid.jsx');
+var Page = require('../../components/page/www/page.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var TTTTile = require('../../components/ttt-tile/ttt-tile.jsx');
+var TTTModal = require('../../components/modal/ttt/modal.jsx');
+var Tiles = require('./ttt.json');
+
+require('./thingstotry.scss');
+
+var ThingsToTry = injectIntl(React.createClass({
+    type: 'ThingsToTry',
+    getInitialState: function () {
+        return {
+            currentTile: Tiles[0],
+            TTTModalOpen: false
+        };
+    },
+    showTTTModal: function (tile) {
+        // expects translated tile
+        this.setState({currentTile: tile});
+        this.setState({TTTModalOpen: true});
+    },
+    hideTTTModal: function () {
+        this.setState({TTTModalOpen: false});
+    },
+    renderTTTTiles: function () {
+        var formatMessage = this.props.intl.formatMessage;
+        var translatedTiles = [];
+        var translatedTile = {};
+
+        Tiles.map(function (tile) {
+            translatedTile = {
+                title: formatMessage({id: tile.title}),
+                description: formatMessage({id: tile.description}),
+                tutorialLoc: tile.tutorialLoc,
+                activityLoc: formatMessage({id: tile.activityLoc}),
+                guideLoc: formatMessage({id: tile.guideLoc}),
+                thumbUrl: tile.thumbUrl,
+                bannerUrl: tile.bannerUrl
+            };
+            translatedTiles.push(
+                <TTTTile
+                    key={tile.title}
```

Comment:
On the off chance that any 2 tiles get the same title, this would raise a warning. The map function retrieves and array ID for the second argument that is good to use for keys like this to avoid that issue. e.g., `function (tile, key) { ... }`